### PR TITLE
Update SC page

### PR DESCRIPTION
**Pull request by riusricardo**

### CODE REVIEW

**Comment by danforbes** [comment]
Closes #586

File: website/pages/en/tutorials.js [hunk 1]
```diff
@@ -36,19 +36,11 @@ const tutorialCardData = [{
...
   title: <translate>Add a Pallet to Your Runtime</translate>,
   text: <translate>Add the Nicks pallet to your Substrate node template.</translate>,
-  difficulty: "medium",
+  difficulty: "easy",
   length: "2",
-  prerequisite: false,
+  prerequisite: true,
   version: "2.0.0",
   href: 'tutorials/add-a-pallet/',
-}, {
```
Comment (danforbes) [inline website/pages/en/tutorials.js:44]: What is the purpose of this change?

File: website/pages/en/tutorials.js [hunk 2]
```diff
@@ -36,19 +36,11 @@ const tutorialCardData = [{
...
-  title: "Upgrade a Chain",
-  text: "Perform a forkless runtime upgrade on a running Substrate network.",
-  difficulty: "medium",
-  length: "2",
-  prerequisite: false,
-  version: "2.0.0",
-  href: 'tutorials/upgrade-a-chain/',
```
Comment (danforbes) [inline website/pages/en/tutorials.js:51]: What is the purpose of this change?

Comment (riusricardo) [reply website/pages/en/tutorials.js:51]: Just re-ordering a little bit based in difficulty.

**Review by danforbes — APPROVED** [review]
